newIDE/app/src/InstancesEditor/index.js

```diff
@@ -182,7 +182,7 @@ export default class InstancesEditor extends Component<Props> {
     this.pixiRenderer.view.onwheel = (event: any) => {
       const zoomFactor = this.getZoomFactor();
       if (this.keyboardShortcuts.shouldZoom()) {
-        this.zoomOnCursorBy((-event.deltaY / 2000) * this.getZoomFactor());
+        this.zoomOnCursorBy(1 - event.deltaY / 100);
```

Comment:
What if `deltaY >= 100`?

Reply:
I'm not sure about that: should we respect the sensitivity set by the user for their mouse or should it be a predefined step?

Reply:
What is the variable for the "sensitivity set by the user"?

Reply:
I think deltaY is the one: https://adom.as/javascript-mouse-wheel/#:~:text=You%20might%20wonder,e.g.%2030.

Reply:
I think that's the quantity of scroll of a wheel event.
A user that tuned their mouse with high sensitivity will maybe have a deltaY of 3 and a user with lower sensitivity something like 0.5.

Reply:
> I think that's the quantity of scroll of a wheel event. A user that tuned their mouse with high sensitivity will maybe have a deltaY of 3 and a user with lower sensitivity something like 0.5.

For a scroll, does it mean applying 3 times the unitary scroll displacement? In this case it's the same as applying the function 3 time with `deltaY = 1`. As zoom are multiplications, applying the zoom (`initialZoom * myFactor`) 3 times would mean doing `initialZoom * myFactor * myFactor * myFactor` which is `initialZoom * myFactor ** deltaY`.